client/components/Slide/Components/CheckResponse/Display.jsx

```diff
@@ -0,0 +1,44 @@
+import { type } from './type';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container, Label, Checkbox } from 'semantic-ui-react';
+
+import './CheckResponse.css';
+
+const Display = ({
+    prompt,
+    checked,
+    unchecked,
+    responseId,
+    onResponseChange
+}) => (
+    <Container text>
+        <Label as="label" className="CheckResponse-label">
+            <Checkbox
+                name={responseId}
```

Comment:
Why do we expose a response ID? What should it be used for? In testing, we found users had this question as well.

Reply:
response ID is like - what the response is in a "form" -- so two questions with the same response id would have the same "answer" -- it could be used for a "response recall" and it's the response "key" in the database

Reply:
Should the Teacher Moment author be able to set the id themselves? Or is this something we can generate based on the prompt? I'm wondering because it adds some complexity to the editor that testers didn't understand. Perhaps we can avoid exposing this?

Reply:
We can definitely generate this automatically, but that's work I didn't want to solve yet - so I exposed setting the response id manually for now